andela/team-odd-bn-frontend
# Improve existing logical operation

#### What does this PR do? 
The PR helps to refactor the existing work.

#### Description of Task to be completed?
- Refactor sidebar.
- Remove <Del>string comparison<Del>
- Add style

#### How should this be manually tested?
- `git clone https://github.com/andela/team-odd-bn-frontend.git`
- `cd team-odd-bn-backend`
- bg-frontend-design-improvement-171403437 
- `npm install`

> Pass wrong <del>token</del> will redirect to `signin`
> Skip editing profile step will redirect you to `/profile` page
> Skip email verification will redirect to `verify-email` page

#### Any background context you want to provide?
The current work works well. However, we need to improve.

#### What are the relevant pivotal tracker stories?
[#171403437](https://www.pivotaltracker.com/story/show/171403437)

#### Screenshots (if appropriate)
#### Questions:
